volontulo/templates/common/footer.html

```diff
@@ -26,6 +26,7 @@
             <h3>Oferty</h3>
             <ul>
                 <li><a href="{% url 'offer_apply' 1 %}">Zgłoś chęć współpracy przy ofercie</a></li>
+                <li><a href="{% url 'contact_form' %}">Kontakt z administratorem</a></li>
```

Comment:
isn't this a duplicate of line 9 in this file?
I think it should be there because it doesn't really fit in offers column